src/Sql/Sql/ChangeLog.md

```diff
@@ -21,6 +21,12 @@
 * Added cmdlet `New-AzSqlServerTrustGroup`
 * Added cmdlet `Remove-AzSqlServerTrustGroup`
 * Added cmdlet `Get-AzSqlServerTrustGroup`
+* Added cmdlet `Copy-AzSqlDatabaseLongTermRetentionBackup`
+    - Copy LTR backups to different servers
+* Added cmdlet `Update-AzSqlDatabaseLongTermRetentionBackup`
+    - Update Backup Storage Redundancy values for LTR backups
+* Added CurrentBackupStorageRedundancy, RequestedBackupStorageRedundancy to `Get-AzSqlDatabase`, `New-AzSqlDatabase`, `Set-AzSqlDatabase`, `New-AzSqlDatabaseSecondary`, `Set-AzSqlDatabaseSecondary`, `New-AzSqlDatabaseCopy`
+    - Changed StorageAccountType value to CurrentBackupStorageRedundancy, RequestedBackupStorageRedundancy to reflect both the current value and what has been requested if a change was made
```

Comment:
Please merge master into the preview branch and resolve conflicts.
The changelog.md should be 

```suggestion
## Upcoming Release

## Version 3.0.0-Preview
* Added cmdlet `Copy-AzSqlDatabaseLongTermRetentionBackup`
    - Copy LTR backups to different servers
* Added cmdlet `Update-AzSqlDatabaseLongTermRetentionBackup`
    - Update Backup Storage Redundancy values for LTR backups
* Added CurrentBackupStorageRedundancy, RequestedBackupStorageRedundancy to `Get-AzSqlDatabase`, `New-AzSqlDatabase`, `Set-AzSqlDatabase`, `New-AzSqlDatabaseSecondary`, `Set-AzSqlDatabaseSecondary`, `New-AzSqlDatabaseCopy`
    - Changed StorageAccountType value to CurrentBackupStorageRedundancy, RequestedBackupStorageRedundancy to reflect both the current value and what has been requested if a change was made
```